internal/stage_br_basic.go

```diff
@@ -0,0 +1,49 @@
+package internal
+
+import (
+	tester_utils "github.com/codecrafters-io/tester-utils"
+)
+
+func testBrBasic(stageHarness *tester_utils.StageHarness) error {
+	testCases := []TestCase{
```

Comment:
Just realized that we don't have randomness in this challenge 😮 Might need to add this later on, guess we can ignore for now